hangupsbot/plugins/slackrtm.py

```diff
@@ -1157,13 +1159,28 @@ def handle_reply(self, reply):
                 if msg.file_attachment:
                     if sync.image_upload:
                         self.loop.call_soon_threadsafe(asyncio.async, self.upload_image(sync.hangoutid, msg.file_attachment))
+                        self.lastimg = os.path.basename(msg.file_attachment)
                     else:
                         # we should not upload the images, so we have to send the url instead
                         response += msg.file_attachment
+                self.sending += 1
                 self.bot.send_html_to_conversation(sync.hangoutid, response)
 
+    @asyncio.coroutine
     def handle_ho_message(self, event):
         for sync in self.get_syncs(hangoutid=event.conv_id):
+            if self.sending and ': ' in event.text:
```

Comment:
This is a pretty horrible way to test for that.  Can we use the message context instead?